src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -423,19 +488,32 @@ public CollectionConfiguration GetCollectionConfig(Collection collection)
         [NotNull]
         internal ReplicatorConfiguration Freeze()
         {
-            var retVal = new ReplicatorConfiguration(Database, Target)
-            {
-                Authenticator = Authenticator,
-                #if COUCHBASE_ENTERPRISE
-                AcceptOnlySelfSignedServerCertificate = AcceptOnlySelfSignedServerCertificate,
-                #endif
-                Continuous = Continuous,
-                PushFilter = PushFilter,
-                PullFilter = PullFilter,
-                ReplicatorType = ReplicatorType,
-                ConflictResolver = ConflictResolver,
-                Options = Options
-            };
+            ReplicatorConfiguration retVal;
+//            if (Collections.Count > 0) {
+//                retVal = new ReplicatorConfiguration(Database, Target)
+//                {
+//                    Authenticator = Authenticator,
+//#if COUCHBASE_ENTERPRISE
+//                    AcceptOnlySelfSignedServerCertificate = AcceptOnlySelfSignedServerCertificate,
+//#endif
+//                    Continuous = Continuous,
+//                    ReplicatorType = ReplicatorType,
+//                    Options = Options,
+//                    CollectionConfigs = CollectionConfigs
+//                };
+//            } else {
```

Comment:
woops, I forgot remove these commented code. Will do it with any request change.